### ISSUE TITLE
reportFilter not excluding PolicyReport from namespaces

### ISSUE THREAD

**Issue by jemag** [fjogeleit/policy-reporter#294]
Specifying reportFilter to exclude specific namespaces does not seem to exclude them from being processed. 
For example in helm chart, specification is currently:
```yaml
  reportFilter:
    namespaces:
      exclude: ["kube-system", "calico-system", "tigera-operator"]
    clusterReports:
      disabled: false
```

Which results in the following config.yaml stored in secret:
```yaml
reportFilter:
  namespaces:
    exclude:
      - kube-system
      - calico-system
      - tigera-operator
  clusterReports:
    disabled: false
```

However, browsing through the UI we can see:
![image](https://user-images.githubusercontent.com/7985687/230148440-0479af9d-a493-42cd-b79c-ff38747a47f3.png)
and
![image](https://user-images.githubusercontent.com/7985687/230148268-a6fb5f6c-d389-4e26-8f92-038d9a953f05.png)

----
Helm: 2.18.1
Kubernetes: 1.24.6

**Comment by fjogeleit** [comment]
Thanks for reporting, I will have a look on it.

**Comment by fjogeleit** [comment]
Because of an syntax error in the config.yaml, it didn't apply everything after this config correctly. This was fixed in #295 

I will release a fixed version today

**Comment by fjogeleit** [comment]
I released the fix with Helm Chart Version 2.18.3
